tests/suite/test_jwt_policies_vsr.py

```diff
@@ -352,7 +352,7 @@ def test_jwt_policy_delete_secret(
         assert f"Request ID:" in resp1.text
         assert crd_info["status"]["state"] == "Warning"
         assert (
-            f'"{v_s_route_setup.route_m.namespace}/{secret}" which does not exist'
+            "references an invalid Secret: secret doesn't exist or of an unsupported type"
```

Comment:
No more secret name in warning message?

Reply:
good point. I can improve those messages in the next PR that implements remaining warnings